src/main/java/org/webcurator/core/harvester/agent/HarvestAgentClient.java

```diff
@@ -237,11 +231,13 @@ public void purgeAbortedTargetInstances(List<String> targetInstanceNames) {
     }
 
     public boolean isValidProfile(String profile) {
+        HttpEntity<String> request = this.createHttpRequestEntity(profile);
```

Comment:
This json-ifies the XML string, so it is no longer well-formed XML and will be rejected by Heritrix. This is probably easiest to fix by simply doing `HttpEntity<String> request = new HttpEntity<String>(profile);`

Reply:
Accepted and fixed as advised.